src/monet.js

```diff
@@ -177,6 +177,22 @@
         return listFindC(l, fn).run()
     }
 
+    function listContainsC(l, val) {
+        if (l.isNil) {
+            return Return(false)
+        }
+        var h = l.head()
+        return h === val ?
```

Comment:
ok i understand now. let's add this, i'll just add comments. too bad we don't (afaict) don't cover arrays though